File: Contest/CodeForces/Contest/1195/d1/d1.py
# -*- coding: utf-8 -*-
# @Author: apikdech
# @Date:   22:57:59 17-07-2019
# @Last Modified by:   apikdech
# @Last Modified time: 23:06:15 17-07-2019
def f(x):
	i = 10
	res = 0
	while(x > 0):
		res += ((int(x % 10)) * i)
		x /= 10
		i *= 100
	return int(res)

n = int(input())
a = list(map(int, input().split()))
ans = 0
mod = 998244353


for x in a:
	tmp = f(x)
	ans += tmp + tmp//10

ans *= n
print(ans % mod)

File: Contest/CodeForces/Contest/1213/d1/d1.cpp
/*
 * Author: apikdech
 * Time: 22:31:16 30-08-2019
**/

#include <bits/stdc++.h>
using namespace std;
typedef long long ll;
const int MOD = 1000000007, N = 55, M = 2e5 + 5;
int a[N], cnt[M], mx = 0;

int dist(int x, int num) {
	int res = 0;
	while(num != x && num > 0) num /= 2, res++;
	if (num == x) return res;
	return -1;
}

int main() {
	ios::sync_with_stdio(0); cin.tie(0);
	int n, k, ans = M;
	cin >> n >> k;
	for (int i = 0; i < n; i++) {
		cin >> a[i];
		cnt[a[i]]++;
		mx = max(mx, a[i]);
	}
	sort(a, a+n);
	for (int i = 0; i <= mx; i++) {
		int tmp = 0, c = 0;
		for (int j = 0; j < n; j++) {
			if (i > a[j]) continue;
			int t = dist(i, a[j]);
			if (t != -1) tmp += t, c++;
			if (c >= k) {
				ans = min(ans, tmp);
				break;
			}
		}
	}
	cout << ans << "\n";
	return 0;
}
